test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1902,18 +1902,18 @@ def test_overlap_resv(self):
         a = {ATTR_queue: r1name[0], 'Resource_List.select': '1:ncpus=1'}
         j1 = Job(TEST_USER, a)
         jid1 = self.server.submit(j1)
-        # Sleeping in between job submission to allow
-        # scheduler pick first job
-        time.sleep(1)
+        self.server.expect(JOB, {'job_state': 'Q'}, id=jid1)
```

Comment:
Just checking for the job in the 'Q' state is not enough to make sure the scheduler has seen the job.  The job will be in the 'Q' state when it is initially submitted.  To make sure the scheduler has seen the job and it remains in the 'Q' state, you need to check that the comment is set as well.